specs/test_7.js

```diff
@@ -0,0 +1,29 @@
+let MainPage = require("../pages/main.page");
+let ProductsCatalogPage = require("../pages/productsCatalog.page");
+let CartPage = require("../pages/cart.page");
+
+describe('Hotline Protractor Test - Work with card suite', () => {
+    it('TC #7. Delete from cart', async () => {
+        await MainPage.open();
+        await MainPage.waitForPageToBeAvailable();
+        await MainPage.navigateToSubSubCategoryPump();
+        await ProductsCatalogPage.waitForPageToBeAvailable();
+        await ProductsCatalogPage.addThreeItemsToCart();
+        await browser.sleep(7000);
```

Comment:
please avoid using `await browser.sleep(7000);`
you should use waiters with expected conditions

Reply:
done